scripts/run_performance_job.py

```diff
@@ -829,18 +925,22 @@ def run_performance_job(args: RunPerformanceJobArgs):
         if args.os_group == "windows":
             bdn_arguments += ["--corerun", f"%HELIX_CORRELATION_PAYLOAD%\\dotnet-mono\\shared\\Microsoft.NETCore.App\\{product_version}\\corerun.exe"]
         else:
-            bdn_arguments += ["--corerun", f"$HELIX_CORRELATION_PAYLOAD/dotnet-mono/shared/Microsoft.NETCore.App/{product_version}/corerun"]
```

Comment:
Was this removal of the + on purpose? It seems like we would want to keep the + as there is some usage of bdn_arguments above.